### PR TITLE
Remove hard limit on number of MPUs in galleries

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
[Galleries - Increase the no of MPU from 2 to more (as many the page will accept based on the no of images )](https://trello.com/c/I7VOxHYp/211-galleries-increase-the-no-of-mpu-from-2-to-more-as-many-the-page-will-accept-based-on-the-no-of-images)

File: static/src/javascripts/projects/commercial/modules/gallery-adverts.js [hunk 1]
```diff
@@ -64,7 +55,15 @@ define([
                 item.anchor.appendChild(item.adSlot);
             }
 
-    });
+        });
+
+    }
+
+    function getSlotNameForMobile(index) {
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/commercial/modules/gallery-adverts.js]: we should have this logic shared somewhere.

**Comment by rich-nguyen** [comment]
👍

File: static/src/javascripts/projects/commercial/modules/gallery-adverts.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/gallery-adverts.js]: Yes! Whilst I was doing the change, I thought exactly the same. This should be in `create-slot`, and naming the slot at the calling site should be optional. Will change next 😃

**Pull request merged**